docs/commands/browser/runStep.mdx

```diff
@@ -0,0 +1,67 @@
+# runStep
+
+## Обзор {#overview}
+
+Используйте команду `runStep`, чтобы получить человекочитаемую историю выполнения теста, которая, в том числе, автоматически будет отображаться в [html-reporter][reporter].
+Шаги могут быть вложенными.
+
+## Использование {#usage}
+
+```javascript
+await browser.runStep(stepName, stepCb);
+```
+
+## Параметры команды {#parameters}
+
+<table>
+<thead>
+<tr><td>**Имя**</td><td>**Тип**</td><td>**Описание**</td></tr>
+</thead>
+<tbody>
+<tr><td>stepName</td><td>String</td><td>Название шага.</td></tr>
```

Comment:
Why is everyone copying this flat weirdly looking table markup with wrong indents, LOL. It wasn't intended, it's a temporary consequence of migrating from yfm. I suggest using proper markup, which will be automatically formatted by prettier e.g. https://github.com/gemini-testing/testplane-docs/pull/26/files

For some reason when tables are formatted this way, Prettier for some reason doesn't pick them up. If you add at least one indent, it will format it automatically.

Applies to all pages